src/services/sdk.ts

```diff
@@ -159,7 +158,7 @@ const buildPublication = (
 const dispatchActivityContent = (
   dispatch: Dispatch,
   message: BroadcastAnnouncement,
-  activityContent: ActivityContent,
+  activityContent: ActivityContentNote | ActivityContentProfile,
   blockNumber: number
 ) => {
   switch (activityContent.type) {
```

Comment:
Non-blocking: the [`isActivityContentNote()`](https://github.com/LibertyDSNP/sdk-ts/blob/main/src/core/activityContent/validation.ts#L173) and [`isActivityContentProfile()`](https://github.com/LibertyDSNP/sdk-ts/blob/main/src/core/activityContent/validation.ts#L200) methods in `sdk/core/activityContent` might save you the trouble of casting the type on lines 169 and 176 below.